source/isaaclab/isaaclab/actuators/actuator_net.py

```diff
@@ -80,7 +80,7 @@ def compute(
         self.sea_input[:, 0, 0] = (control_action.joint_positions - joint_pos).flatten()
         self.sea_input[:, 0, 1] = joint_vel.flatten()
         # save current joint vel for dc-motor clipping
-        self._joint_vel[:] = joint_vel
+        self._joint_vel[:] = torch.clip(joint_vel, min=-self.velocity_limit, max=self.velocity_limit)
```

Comment:
We will need to somehow verify our sim-to-real for the robots if all these changes happen. Even if they do make sense mathematically, they may affect downstream sim-to-real pipelines unknowingly.

Is there a way to verify this on our side first before rolling it out to the public?